nevergrad/functions/control/core.py

```diff
@@ -0,0 +1,163 @@
+    state_std = [0.09823861308889045, 0.17141367557864626, 0.0923065066441211, 0.09857913658212542, 0.23067414103638456, 0.31929625189221916, 0.2789541347319086, 0.4252996376350357, 0.17273612261821686, 0.3024306630085608, 0.15440628974854603, 0.42709280554821294, 0.21228009754109947, 1.627223228431405, 0.8187247777350416, 0.9552450884883069, 0.9842769566724734, 1.1630255246361634, 0.9579164107073792, 2.1506606935604275, 3.602875428572795, 5.628269094686002, 0.9920290672268143, 2.1223003669071345, 1.0003529831706024, 5.57262446791663, 2.12884125072627, 1e-08, 1e-08, 1e-08, 1e-08, 1e-08, 1e-08, 0.04912611953512608, 0.04889164520640186, 0.010828667552457222, 0.07233078284166466, 0.07267150019759298, 0.0909353971096799, 1e-08, 1e-08, 1e-08, 1e-08, 1e-08, 1e-08, 0.015251132328387708, 0.01519859897632306, 0.014332713037703749, 0.016533211312666475, 0.01655178210918886, 0.017826147794067864, 0.19608990202601936, 0.20036636045336445, 0.1867258748179935, 0.19829593162153397, 0.19688339050234013, 0.20454779790272495, 1e-08, 1e-08, 1e-08, 1e-08, 1e-08, 1e-08, 0.018432894986970972, 0.018497451738447097, 0.01722172494351193, 0.020457389878712977, 0.020483738957125595, 0.022133816429764575, 0.3350906849993091, 0.33250359129341234, 0.33900096746510755, 0.3443797803052648, 0.3439163943012149, 0.33448413391786447, 1e-08, 1e-08, 1e-08, 1e-08, 1e-08, 1e-08, 0.019536561966571657, 0.0193568339032099, 0.0178721598998528, 0.0215286944065503, 0.021618066835186175, 0.023317064797031488, 0.28256515904161045, 0.2805053208089945, 0.2749447000464192, 0.2834385598495182, 0.2828637149337391, 0.2845542953486725, 1e-08, 1e-08, 1e-08, 1e-08, 1e-08, 1e-08, 0.01632940767643436, 0.01639546342576569, 0.015535938645943628, 0.018094969304990823, 0.01806480253410683, 0.019505571384448334, 0.3051507639463494, 0.30417415230423156, 0.29980481621099997, 0.30983874726354155, 0.3104093749664208, 0.30598187568861357]
+
+    def __init__(self, num_rollouts: int, random_state: tp.Optional[int] = None) -> None:
+        super().__init__(self._simulate_ant, p.Array(shape=(888,)))
```

Comment:
shape is deduced from policy_dim right? can it be directly policy_dim (as a 2d array)? It looks like it is put back in shape later on, so this way that would remove the unnecessary reshape and keep the structure of the problem.

Reply:
It seems that input of ``__simulate`` is always 1D even if parameter was initiated with 2D shape. How can I enforce parameter to keep initial shape ?

Reply:
Yes, I double checked it. I'll also investigate it on my part.

Reply:
Ok I think I've got it:
in `_simulate` you will get as input whatever you provide to the callable. It does not reshape the input in anyway.
However, because you provided the right shape to the parametrization, the inputs created through the parametrization will have the right shape.

So the problem is not really that it's wrong, it's that the test is not testing the right thing. You should be testing `func(func.parametrization.value)` in the test for instance.
You can keep the reshape anyway since it eases the tests, but it's up to you.

Reply:
Thank you very much !